modules/planning/tasks/deciders/path_bounds_decider.h

```diff
@@ -74,17 +74,18 @@ class PathBoundsDecider : public Decider {
   SortObstaclesForSweepLine(
       const IndexedList<std::string, Obstacle>& indexed_obstacles);
 
-  int ConstructSubsequentPathBounds(
+  std::vector<std::vector<std::tuple<double, double, double>>>
+  ConstructSubsequentPathBounds(
       const std::vector<std::tuple<int, double, double, double, std::string>>&
       sorted_obstacles,
       size_t path_idx, size_t obs_idx,
       std::unordered_map<std::string, std::tuple<bool, double>>*
       const obs_id_to_details,
-      std::vector<std::tuple<double, double, double>>* const curr_path_bounds,
-      std::vector<std::tuple<double, double, double>>* const final_path_bounds);
+      std::vector<std::tuple<double, double, double>>* const curr_path_bounds);
 
   std::vector<std::vector<bool>> DecidePassDirections(
-      std::vector<std::tuple<int, double, double, double, std::string>>* const
+      const double& l_min, const double& l_max,
```

Comment:
No need to pass reference on primitive types. Simply pass value.

Reply:
fixed this.